rdkit/Chem/Draw/__init__.py

```diff
@@ -476,8 +476,11 @@ def _moltoimg(mol, sz, highlights, legend, returnPNG=False, drawOptions=None, **
     # we already prepared the molecule:
     d2d.drawOptions().prepareMolsBeforeDrawing = False
     bondHighlights = kwargs.get('highlightBonds', None)
-    d2d.DrawMolecule(mc, legend=legend or "", highlightAtoms=highlights or [],
-                     highlightBonds=bondHighlights or [])
+    if bondHighlights is not None:
```

Comment:
What are the actually differences here, I would have expected the same results.

Reply:
It ends up calling a different method on the C++ side

Reply:
Yep, just goes to show don't fix what ain't broken.